dao/OrderDao.py

```
@@ -0,0 +1,267 @@
+            master_sql = sql.SQL("""
+                INSERT INTO orders (
+                    user_id, 
+                    total_amount,
+                    status,
+                    order_number,
+                    created_at
+                ) VALUES (%s, %s, %s, %s, NOW()) RETURNING order_id
+            """)
+            
+            # 生成订单号（实际应用中应使用更复杂的规则）
+            order_number = f"ON{int(time.time())}{random.randint(1000, 9999)}"
```

Ruff (0.11.9)

Undefined name `time`

(F821)